packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -105,3 +118,31 @@ function buildCQLinks(pdResults: OutboundPatientDiscoveryResp[]): CQLink[] {
     };
   });
 }
+
+async function augmentPatientDemographics(
+  patient: Pick<Patient, "id" | "cxId">,
+  pdResults: OutboundPatientDiscoveryResp[]
+): Promise<boolean> {
```

Comment:
Do we want to limit the amount of times we augment Patient demographics to the first PD, or do we want to execute it on each PD?